policies/elections.tex

```diff
@@ -315,12 +328,12 @@ \subsubsection{During Election}
 The Elections Committee may also take up violations of its own initiative.
 For violation of the rules, the Elections Committee may impose a reduction of spending limit, order removal of campaign material, limit a candidate's means of campaigning, or impose other penalties as it sees fit, except that it may disqualify candidates only when so authorized.
 
-A candidate who overspends or fails to submit an expense report shall be disqualified by the Elections Committee.
+A candidate who overspends or fails to submit an expense report shall be disqualified by the Elections Committee, unless that candidate did not spend any money on their campaign.
```

Comment:
A candidate who overspends or fails to submit an expense report within a reasonable timeframe shall be disqualified by the Elections Committee.